packages/react-components/react-skeleton/src/components/SkeletonCircle/useSkeletonCircleStyles.ts

```diff
@@ -1,33 +1,43 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { SkeletonCircleSlots, SkeletonCircleState } from './SkeletonCircle.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
 
 export const skeletonCircleClassNames: SlotClassNames<SkeletonCircleSlots> = {
-  root: 'fui-SkeletonCircle',
-  // TODO: add class names for all slots on SkeletonCircleSlots.
-  // Should be of the form `<slotName>: 'fui-SkeletonCircle__<slotName>`
+  root: 'fui-Skeleton-Circle',
 };
 
 /**
  * Styles for the root slot
  */
-const useStyles = makeStyles({
+const useRootStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
-  },
+    ...shorthands.borderRadius('50%'),
+    ...shorthands.borderColor(tokens.colorNeutralStencil1),
 
-  // TODO add additional classes for different states and/or slots
+    '@media screen and (forced-colors:active)': {
+      ...shorthands.borderColor('Window'),
+    },
+  },
 });
 
 /**
  * Apply styling to the SkeletonCircle slots based on the state
  */
 export const useSkeletonCircleStyles_unstable = (state: SkeletonCircleState): SkeletonCircleState => {
-  const styles = useStyles();
-  state.root.className = mergeClasses(skeletonCircleClassNames.root, styles.root, state.root.className);
+  const { radius } = state;
+
+  const rootStyles = useRootStyles();
+
+  state.root.className = mergeClasses(skeletonCircleClassNames.root, rootStyles.root, state.root.className);
 
-  // TODO Add class names to slots, for example:
-  // state.mySlot.className = mergeClasses(styles.mySlot, state.mySlot.className);
+  if (radius) {
+    state.root.style = {
+      height: radius,
+      width: radius,
```

Comment:
For me it's strange that radius defines `height` and `width`:

![image](https://user-images.githubusercontent.com/14183168/216949581-8eee7f54-6f20-4c59-9e59-eb17102dadfe.png)

Should it be "diameter" instead (_radius is a half of diameter_)? Or some other name...

Reply:
I agree it does seem counterintuitive to use radius, but that is the prop name most commonly used to determine a circle's dimensions, so it is the one I went with

Reply:
I see, but I still suggest to consider to use a different name as current implementation is counterintuitive:

![image](https://user-images.githubusercontent.com/14183168/217200612-c55d7b4f-48b0-41ea-8f2d-b00fe02632ee.png)

Reply:
`radius` has been changed to `size`